platforms/ios/framework/src/TGMapView.mm

```diff
@@ -1220,6 +1220,21 @@ - (void)notifyGestureDidEnd
 
 #pragma mark Internal Logic
 
+- (void)flyToCameraPosition:(TGCameraPosition *)cameraPosition
+               withDuration:(NSTimeInterval)duration
+                      speed:(CGFloat)speed
+                   callback:(void (^)(BOOL))callback
+{
+    Tangram::CameraPosition camera = [cameraPosition convertToCoreCamera];
+    if (duration == 0) {
+        [self setMapRegionChangeState:TGMapRegionJumping];
```

Comment:
Feels weird to bifurcate this state behavior based on duration. Can't think of any practical problems with it though.

Reply:
The practical affect is actually "JUMPING" so I think it does make sense.